tests/utils/blockchain.py

```diff
@@ -1,4 +1,5 @@
 import os
+from ape_test import LocalProvider
```

Comment:
This doesn't appear to be used (yet) - is there a reason for that?

Reply:
if it's unused feel free to remove it. I think I was using it as a type hint in an earlier iteration.